include/caffe/util/upgrade_proto.hpp

```diff
@@ -16,8 +16,12 @@ bool UpgradeNetAsNeeded(const string& param_file, NetParameter* param);
 // Read parameters from a file into a NetParameter proto message.
 void ReadNetParamsFromTextFileOrDie(const string& param_file,
                                     NetParameter* param);
+void ReadNetParamsFromTextStreamOrDie(std::istream& param_stream, const string& name_to_log,
```

Comment:
& to the right

Reply:
> & to the right

This is just an extension to caffe code. We have to use caffe coding style.